### DISCUSSION TITLE
Link to "in development standards" on doc site is broken

### DISCUSSION THREAD

**Discussion by riziles** [fastapi/sqlmodel#788] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```markdown
SQLModel even adopts currently 
[in development standards](https://github.com/microsoft/pyright/blob/main/specs/dataclass_transforms.md)
for Python type annotations to ensure the best developer experience, so you will
get inline errors and autocompletion even while creating new model instances.
```

### Description

The link on the doc site ["features"](https://sqlmodel.tiangolo.com/features/) page referencing [in development standards](https://github.com/microsoft/pyright/blob/main/specs/dataclass_transforms.md) returns a 404. I **think** that it should point here instead: https://peps.python.org/pep-0681/ .

### Operating System

Windows

### Operating System Details

This issue is not OS specific.

### SQLModel Version

0.0.14

### Python Version

3.10

### Additional Context

The docs site is amazing. I just want to help make it better.

**Comment by YuriiMotov** [comment]
Fixed
